### ISSUE TITLE
Said in the docs to add the Softmax Activation function but never did in code.

### ISSUE THREAD

**Issue by alih552** [tensorflow/tensorflow#45801]
## URL(s) with the issue:

https://www.tensorflow.org/tutorials/images/cnn#add_dense_layers_on_top

## Description of issue (what needs changing):
In the last sentence "CIFAR has 10 output classes, so you use a final Dense layer **with 10 outputs and a softmax activation."** 
"and a softmax activation" **but I guess some one forgot to add the softmax activation at the end.**
**OLD CODE:**
```
model.add(layers.Flatten())
model.add(layers.Dense(64, activation='relu'))
model.add(layers.Dense(10))
```

**NEW CODE:**
```
model.add(layers.Flatten())
model.add(layers.Dense(64, activation='relu'))
model.add(layers.Dense(10))
model.add(layers.Activation('softmax'))
```

**Comment by ymodak** [comment]
Closing this issue since PR has been merged. Thank you.
